CMakeLists.txt

```diff
@@ -141,6 +141,21 @@ endif(WANT_EIO AND THREADS_FOUND AND LIBEIO_FOUND)
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/include/evfibers/config.h.in"
 	"${CMAKE_CURRENT_BINARY_DIR}/include/evfibers/config.h")
 
+# provide pkg-config file
+if(NOT WIN32 AND NOT APPLE)
+  configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/cmake/libevfibers.pc.in
+    ${CMAKE_CURRENT_BINARY_DIR}/CMakeFiles/libevfibers.pc @ONLY
```

Comment:
IIRC `CMakeFiles` is a private CMake folder. It would just work if we put the `.pc` file into `${CMAKE_CURRENT_BINARY_DIR}`.